docs/specs.md

```diff
@@ -145,6 +150,24 @@ an example below:
 
 [<img src="../test_status.png" width="1253px" />](test_status.png)
 
+### Output String Formatting
+
+You can add rich formatting to your autograder output to further
+customize your autograder experience for students.
+
+
+Options for the the format field are as follows:
+
+* `"text"` (default for test output and names) - This will do the
+basic text formatting which was previously done on all test names
+and outputs. No html, markdown, or ansi will be rendered.
+* `"simple_format"` (default for top level output) - This is very
+similar to the `"html"` format option but will also convert `\n`
+into `<br />` and `\n\n+` into a page break.
+* `"html"` - This will render html in your output. Note that we sanitize out
+so you may not be able to use all of the normal html elements!
```

Comment:
```suggestion
* `"html"` - This will render html in your output. Note that we sanitize the
output so you may not be able to use all html elements! Additionally, not all
tags will be styled out of the box.
```